### PR TITLE
build: git: ignore runtime-generated test files

### PR DESCRIPTION
**Pull request by tsiq-clemens**
dealing with a few files that tend to hang around and confuse me when I prep commits

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -7,6 +7,8 @@ node_modules
 webtau.*report*.html
 webtau-report-*.html
 *-webtau-report.html
+webtau.graphql-report.json
+run-details.actual.json
```
Comment (MykolaGolubyev) [inline .gitignore:11]: this should go away when test is passes, so maybe a good idea to have it not ignored. dunno.

**Comment by MykolaGolubyev** [comment]
I would rename PR to `git: `. `core: ` is for actual core logic

File: .gitignore (hunk 1 above)
Comment (tsiq-clemens) [reply .gitignore:11]: you don't want to add it to git, though, right?

Comment (MykolaGolubyev) [reply .gitignore:11]: 50/50. Add it. No harm. Was thinking could be a good signal to remind you to rerun a test

Comment (tsiq-clemens) [reply .gitignore:11]: it is more prominent, I give you that. I'll remove it from here.

**Review by MykolaGolubyev — APPROVED** [review]

**Comment by tsiq-karold** [comment]
> I would rename PR to `git: `. `core: ` is for actual core logic

I think I've used `build:` for this sort of thing, bit broader than git.

**Pull request merged**